### DISCUSSION TITLE
Can I use utterances  on my pages?

### DISCUSSION THREAD

**Discussion by lonr** [community/community#21830] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://utteranc.es/" rel="nofollow noopener">utterances demo page</a></p>
<p><a href="https://github.com/utterance" rel="nofollow noopener">utterances on GitHub</a></p>
<p>It’s said to be a lightweight comments widget using GitHub issues and Styled with <a href="http://primercss.io/" rel="nofollow noopener">Primer</a>(the css toolkit that powers GitHub).</p>
<p>Is its user interface in breach of the terms of GitHub?</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p><a class="mention" href="/lonr">@lonr</a> In order to help best with this particular question, can you please email us at <a href="http://git.io/c" class="inline-onebox-loading" rel="nofollow noopener">git.io/c</a> ? From there, we’ll be able to help a bit better. Thank you!</p>

**Comment by jdanyow** [comment]
<p><a class="mention" href="/nadiajoyce">@nadiajoyce</a> <a class="mention" href="/lonr">@lonr</a> utteranc.es developer here :slight_smile:</p>
<p>What was the response to <a class="mention" href="/lonr">@lonr</a>’s question?</p>
